lib/generateBuffer.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const fs = require('fs');
+const schema = require('protocol-buffers-schema');
+const Pbf = require('pbf');
+const Compile = require('pbf/compile');
+
+const MVT_SPEC_VERSIONS = ['1.0.0', '1.0.1', '2.0', '2.1'];
```

Comment:
This should ideally be dynamic. Can you think of a way to read these dynamically to avoid hardcoding?

Reply:
done!